examples/step-51/doc/intro.dox

```diff
@@ -29,13 +29,12 @@ for systems of equations in 2 or 3 dimensions.
 <h4> Reducing the size of the linear system </h4>
 To alleviate the computational cost of solving such large linear systems,
 the hybridizable discontinuous Galerkin (HDG) methodology was introduced
-by Cockburn and co-workers
-(see the references in the recent HDG overview article by N.C. Nguyen and J. Peraire @cite Ngu2012).
-The HDG method achieves
-this goal by formulating the mathematical problem using Dirichlet-to-Neumann
-mappings.  The partial differential equations are first written as a first
-order system, and each field is then discretized via a DG method.  At this
-point, the  single-valued "trace" values on the skeleton of the
+by Cockburn and co-workers (see the references in the recent HDG overview 
+article by Nguyen and Peraire @cite Ngu2012).
+
+The HDG method achieves this goal by formulating the mathematical problem using
+Dirichlet-to-Neumann mappings.  The partial differential equations are first
+ written as a first order system, and each field is then discretized via a DG method.  At this point, the  single-valued "trace" values on the skeleton of the
```

Comment:
Can you break this line at around 80 characters? I can't actually see what's beyond that point when looking at the pull request :-)

Reply:
Sure, I think I've cut all the super long lines. It should be better now.